src/components/ProjectList.js

```
@@ -0,0 +1,356 @@
+      switch(filters.sortBy) {
+        case 'newest':
+          return new Date(b.created_at || 0) - new Date(a.created_at || 0);
+        case 'oldest':
+          return new Date(a.created_at || 0) - new Date(b.created_at || 0);
+        case 'rank':
+          return (b.rank || 0) - (a.rank || 0);
+        case 'title':
+          return (a.title || '').localeCompare(b.title || '');
+        case 'needHelp':
+          // Projects needing help (no helpers or few helpers) come first
+          const aHelpers = a.helping?.length || 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)